mypy_boto3_builder/stubs_static/types-aiobotocore/session.pyi

```diff
@@ -17,8 +17,8 @@ class ClientCreatorContext:
     async def __aenter__(self) -> AioBaseClient: ...
     async def __aexit__(
         self,
-        exc_type: Optional[Type[Exception]],
-        exc_val: Optional[Exception],
+        exc_type: Optional[Type[BaseException]],
+        exc_val: Optional[BaseException],
```

Comment:
Should be moved to https://github.com/youtype/types-aiobotocore/blob/main/aiobotocore-stubs/session.pyi